release-notes/v2.5.0.md

```diff
@@ -0,0 +1,38 @@
+# HPE CSI Driver for Kubernetes 2.5.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.5.0
+
+| **Version:** |2.5.0|
+|--------------|-----|
+| **Revision:** | Thursday July 25, 2024 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.5.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-648|csi.k8s|Implement topology in the CSI driver|CSI supports the concept of topology by providing the mechanism to define availability zones and associate cluster nodes to those zones.|
+|CON-2508|csi.k8s|Post reboot and drained worker node is not cleaned up properly, unless flush it out|Safely removing or cleaning the stale mount paths or devices attached to a worker node.|
+|CON-3071|csi.k8s|Node monitor support|Node monitor running on every cluster node make sure that the node is healthy.|
```

Comment:
Making sure the node is healthy might be misleading, we can specify something like "make sure no stale multipath entries exist"

Reply:
Michael corrected the resolution note for this and those changes are merged here now.